### ISSUE TITLE
Spatial Dropout

### ISSUE THREAD

**Issue by benkamphaus** [originrose/cortex#56]
I've observed in Keras that `SpatialDropout2D` seems to have a fairly large impact on regularization effectiveness for convolutional layers with less impact on training time. I haven't dug deeply yet, but the paper describing spatial dropout can be found [here](https://arxiv.org/pdf/1411.4280.pdf). The details on Spatial Dropout are in section 3.2.
